Verifier.ts

```diff
@@ -36,43 +39,56 @@ export class Verifier extends Actor<Verifier> {
 						result =
 							splitted.length == 3 &&
 							algorithm &&
-							(await algorithm.verify(`${splitted[0]}.${splitted[1]}`, splitted[2]))
+							(await algorithm.some(async a => await a.verify(`${splitted[0]}.${splitted[1]}`, splitted[2])))
 								? result
 								: undefined
 					}
 				} catch {
 					result = undefined
 				}
-				const now = Date.now()
+				if (result) {
+					//Legacy problem, transform epoch in ms to epoch in s
+				}
```

Comment:
Have you created an issue for solving this?

Reply:
I forgot to cleanup the comments here. This issue i fixed shortly after finding it, it has been resolved in the commit.